rust/psibase/src/graph_ql.rs

```diff
@@ -374,3 +378,284 @@ pub trait NamedEvent {
 pub trait EventDb {
     fn db() -> DbId;
 }
+
+#[derive(Debug)]
+struct SqlRow {
+    rowid: i32,
+    data: Value,
+}
+
+impl<'de> Deserialize<'de> for SqlRow {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        let mut map: HashMap<String, Value> = HashMap::deserialize(deserializer)?;
```

Comment:
Can this use `serde_json::Map` directly and avoid the extra conversion?

Reply:
Yep, good call - done